.github/workflows/reviewdog.yaml

```
@@ -62,6 +62,18 @@ jobs:
       - name: Set Git config
         run: |
           git config --global --add safe.directory ${GITHUB_WORKSPACE}
+      - name: Install sccache
+        run: cargo install sccache --locked
+      - name: Cache sccache
+        uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)